node-manager/src/node.rs

```diff
@@ -288,9 +275,11 @@ impl Node {
                     background_processor_peer_manager.clone(),
                     background_processor_logger.clone(),
                     Some(scorer.clone()),
-                    |d| async move {
-                        sleep(d.as_millis() as i32).await;
-                        false
+                    |d| {
+                        Box::pin(async move {
+                            sleep(d.as_millis() as i32).await;
+                            false
+                        })
```

Comment:
sometimes i see this error that I don't remember really seeing before: 

```
node_manager.js:1681 panicked at '`async fn` resumed after completion', node-manager/src/node.rs:279:34
```

Reply:
yeah needed the Box::pin to make it compile, not sure if that was correct or not

Reply:
this was fixed with https://github.com/MutinyWallet/mutiny-node/pull/328/commits/10f2353fea9cb2071a7f14cec75f75230914bbc1 and LDK is merging it upstream